Bug report template: various improvements (#1997)
WordPress/WordPress-Coding-Standards: .github/ISSUE_TEMPLATE/bug_report.md

---

<!--
PLEASE FILL OUT THE TEMPLATE COMPLETELY.

> ISSUES WHICH CANNOT BE REPRODUCED BASED ON THE INFORMATION PROVIDED WILL BE CLOSED WITHOUT HESISTATION.

Reviewer: ```suggestion
ISSUES THAT CANNOT BE REPRODUCED BASED ON THE INFORMATION PROVIDED WILL BE CLOSED AS UNACTIONABLE.
```

Author: What about simplifying it to:
```diff
+ISSUES WHICH CANNOT BE REPRODUCED BASED ON THE INFORMATION PROVIDED WILL BE CLOSED WITHOUT HESISTATION.
-BUG REPORTS WHICH CANNOT BE REPRODUCED BASED ON THE INFORMATION PROVIDED WILL BE CLOSED.
```

-->

## Minimal Code Snippet

> <!-- Please provide example code that allows to reproduce the issue. Do NOT paste screenshots of code! -->

Reviewer: ```suggestion
<!-- Please provide example code that allows anyone to reproduce the issue. Do NOT paste screenshots of code! -->
```

Author: Well, _anyone_ may be expecting a bit much... ?

Reviewer: Feel free to change to "us". :-)
